recipes/solr-deploycore.rb

```diff
@@ -63,7 +63,8 @@
 	command "unzip -u -q -o #{Chef::Config[:file_cache_path]}/solr_core_config.zip -d #{solr_core_dir}"
 end
 
-service "solr start" do
-    service_name "solr"
-	action [:start]
+execute "solr start" do
+	user 'root'
+	# Try both initd and systemd styles
+	command "(systemctl status solr >/dev/null 2>&1 || service solr status >/dev/null 2>&1) || systemctl start solr"
```

Comment:
we try systemctrl or service for status but only systemctl or starting? 

@chris-randall-qol  are you able review how our init platform works see if there is any improves you can provided based on your learnings in the clean room systemd space. 

Since we should be able to have a test to see if the os is initd or systemd enabled and just use one flavour vs this odd fall back.

info page i found: https://serverfault.com/questions/690155/whats-the-easiest-way-to-make-my-old-init-script-work-in-systemd

Reply:
From what I can see, the initd script will detect any running Solr instance, but Systemd only knows about Solr instances that it personally started.

I suppose I could simplify it so that it only checks 'service solr status' before deciding to start it.